### PR TITLE
Feature/develop: Provider Profile Page

### PR DESCRIPTION
**Pull request by sence1-greniel**
---

Name: "Feature Request ID: 000"
About: _Feature/develop: Provider Profile Page_
Labels: feature
Notion Link: notion_link

---

---

#### Feature Description
 - Create Provider Profile Page (Side Bar Card, Content Card)
 - Integrate Profile Provider Details 
 - Integrate Functionality Subscribe / Unsubscribe Newsletter

---

#### Steps to Reproduce the Feature
- Access Provider Profile Page
- http://localhost:3000/provider/account/1/view-profile
- http://localhost:3000/provider/account/1/account-profile
- http://localhost:3000/provider/account/1/account-security
- http://localhost:3000/provider/account/1/account-settings
- http://localhost:3000/provider/account/1/close-account

http://localhost:3000/provider/account/1 - Results to 404 Page
http://localhost:3000/provider/account - Results to 404 Page
---

#### Expected Behavior
 - Populate data to the contents from the backend.
 - Clicking the sidebar should show correct contents
 - Initial Subscribe/Unsubscribe should work

---

#### Test Environment
 - Browser: [e.g. Chrome, Safarri, Firefox]
 - Test Scenarios:
   - [ ] Did the Test Feature Pass?
   - [ ] Did you check the features on different browsers?

---

#### Additional Details / Context

### CODE REVIEW

File: frontend/src/containers/ProfilesPage/ProviderProfile/ProviderProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    gap: 3
+  },
+  simpleBoxStyle: {
+    width: '100%',
+    my: 10,
+  }
+}
```
Comment (sence1roandayne) [inline frontend/src/containers/ProfilesPage/ProviderProfile/ProviderProfile.tsx]: Can we add all `const theme = {}` related to provider profile page to a different file?

**Review by sence1roandayne — CHANGES_REQUESTED** [review]

**Comment by sence1roandayne** [comment]
- [x] Please add Profile in navigation bar to have access to the profile page. 
- [x] Make `/view-profile` as the lastRoute's initial data.

File: frontend/src/containers/ProfilesPage/ProviderProfile/ProviderProfile.tsx (hunk 1 above)
Comment (sence1-greniel) [reply frontend/src/containers/ProfilesPage/ProviderProfile/ProviderProfile.tsx]: @sence1roandayne thank you for checking roan, i've added moved the styles to the styles with the name profileTheme and implemented the styles to all profile secction

**Review by sence1roandayne — APPROVED** [review]
LGTM

**Pull request merged**